packages/@react-aria/interactions/src/PressResponder.tsx

```diff
@@ -56,7 +56,7 @@ export const PressResponder = React.forwardRef(({children, ...props}: PressRespo
   );
 });
 
-export function ClearPressResponder({children}: {children: ReactNode}): ReactNode {
+export function ClearPressResponder({children}: {children: ReactNode}): JSX.Element {
```

Comment:
and this one https://app.unpkg.com/@react-aria/interactions@3.24.1/files/dist/types.d.ts#L98